types/lodash/lodash-tests.ts

```diff
@@ -96,7 +96,7 @@ interface TResult {
 }
 
 // _.MapCache
-let testMapCache: _.MapCache;
+let testMapCache: _.MapCache = {} as any;
```

Comment:
Unfortunately, because I enabled `strictNullChecks`, I got many errors in the tests due to using uninitialized values. I fixed the errors by initializing them to trivial values.